python/src/timeSeries/timeSeries.py

```diff
@@ -26,8 +26,7 @@
 DXDYDZ = 'dXdYdZ'
 ENU = 'ENU'
 
-########################################################################
-class timeSeries:
+class TimeSeries:
```

Comment:
I like this change. I'm pretty sure in python 'best practices' classes should start with a capital letter. Thanks for catching that.